src/testdir/test_gettext_utf8.vim

```diff
@@ -14,14 +14,28 @@ func Test_gettext()
 
     try
       language messages ru_RU
-      call assert_equal('ОШИБКА: ', gettext("ERROR: ", "__PACKAGE__"))
+      call assert_equal('ОШИБКА: for __PACKAGE__', gettext("ERROR: ", "__PACKAGE__"))
+
+      call assert_equal('ОШИБКА: for __PACKAGE__', ngettext("ERROR: ", "ERRORS: ", 1, "__PACKAGE__"))
+      call assert_equal('ОШИБКА: for __PACKAGE__', ngettext("ERROR: ", "ERRORS: ", 2, "__PACKAGE__"))
+
+      call assert_equal('%d буфер удалён из памяти for __PACKAGE__', ngettext("%d buffer unloaded", "%d buffers unloaded", 1, "__PACKAGE__"))
+      call assert_equal('%d буфера удалено из памяти for __PACKAGE__', ngettext("%d buffer unloaded", "%d buffers unloaded", 2, "__PACKAGE__"))
+      call assert_equal('%d буферов удалено из памяти for __PACKAGE__', ngettext("%d buffer unloaded", "%d buffers unloaded", 5, "__PACKAGE__"))
```

Comment:
Same here.
https://github.com/vim/vim/pull/16529/commits/dab369a90f270b01be8e7fb85b70083538056686#diff-f1f42e08d5b4e0d62dec4c8994afcf28757c584e8daa69eaa579cc80aa9965df

Reply:
Checked it again, it's fine. There must be some glitch in Chromium